proposals/0441-formalize-language-mode-terminology.md

```diff
@@ -49,7 +49,7 @@ Add a new `init` method to `Package` with the following changes from the current
 - The parameter `swiftLanguageVersions` is renamed to `swiftLanguageModes`
 - The parameter type is now an optional array of `SwiftLanguageMode` values instead of `SwiftVersion` values
 
-The existing init method will be marked as `obsoleted` and `renamed` allowing the compiler to provide a fix-it.
+The existing init method will be marked as `deprecated` and `renamed` allowing the compiler to provide a fix-it. It will also have the @_disfavoredOverload attribute added to allow the compiler to disambiguate.
```

Comment:
```suggestion
The existing init method will be marked as `deprecated` and `renamed` allowing the compiler to provide a fix-it.
```

Reply:
Change made in 540b9c6c77a48fefd68ca248daba749b6ed22875.